### DISCUSSION TITLE
Configuration Nucleo_u575zi_q( BLE Beacon ) board with mobile devices

### DISCUSSION THREAD

**Discussion by VinayShirol-eaton** [zephyrproject-rtos/zephyr#56102] [Q&A]
**Describe the bug**
Hi, I am using nucleo_u575zi board with Murata 1DX chip ( Act as BLE and WIFI ) . Coming to my bug i am using ibeacon sample available ( Zephyr/Samples/Bluetooth/Ibeacon) i can successfully build and flash the code and testing in BLE beacon scanners is also successfully but I need to test in mobile ( iOS & android ) . it is not happening 

Hardware and Platform Reguirements
1.  Zephyr Platform
2. Murata 1DX Chip
3. Nucleo_U575zi_q
4. gnu-arm-toolchain

**Expected behavior**
1. As of now we are testing ibeacon with other BLE Scanners
2. But we need to test on mobile devices ( IOS & ANDROID )
3. The configuration of BLE Ibeacon to Mobile Device is not happening
4. [https://www.youtube.com/watch?v=r-AR2PsNVOM](url)
Please refer that youtube link we are expecting to work like that

**Impact**

Our project got struck here beacuse we dont know how zephyr will test BLE Ibeacon on Mobile Devices

**Environment (please complete the following information):**
 - Windows: (e.g. Linux, MacOS, Windows)
 - GNU-ARM-TOOLCHAIN
 - Zephyr 3.2.0

**Comment by erwango** [comment]
Since you're not testing a configuration which is provided available in tree, please provide more information on the project you're building (git branch for instance) so people can have a look to the configuration you're using.

**Comment by jgl-meta** [comment]
This is not really a bug. We will leave this unassigned for now.

**Comment by erwango** [comment]
> This is not really a bug. We will leave this unassigned for now.

@jgl-meta I propose to move this to discussion then, until the expectation is clarified.
